src/js/components/Sidebar.js

```diff
@@ -364,14 +337,6 @@ class Sidebar extends React.Component {
               </div>
             </div>
           </GeminiScrollbar>
-          <div className="sidebar-footer">
```

Comment:
CSS classes need to removed and tests need to be updated for `sidebar-footer`

Reply:
Did you mean CSS code?

Reply:
Yes, code was removed.